app/controllers/organizations_controller.rb

```diff
@@ -1,17 +1,18 @@
 class OrganizationsController < ApplicationController
   before_action :authenticate_user!
+  load_and_authorize_resource
   layout 'org_admin'
 
   def show
     @org = Organization.friendly.find(params[:id])
     session[:current_campaign] ||= Campaign.current_campaign_id(params[:id])
-    if session[:current_campaign]
+    if session[:current_campaign] == -1
+      @organization_campaign = nil
+    else
       @organization_campaign = OrganizationCampaign.where(
         organization_id: @org.id,
         campaign_id: session[:current_campaign]
       ).first
-    else
-      @organization_campaign = nil
     end
     @joined_campaigns = Campaign.joined_by(@org.id)
```

Comment:
Would it make sense to move these two calls as methods or scopes on the `Organization` model as well?

```
o = Organization.find(params[:id])
o.joined_campaigns # campaigns already joined
o.joinable_campaigns # campaigns able to join
```

It seems like we might be missing out on some of the ActiveRecord goodness by hitting the `Campaign` model directly, versus leveraging the `Organization => OrganizationCampaigns => Campaign` relationship we've established